src/DesignKit.tsx

```diff
@@ -3588,47 +3851,76 @@ const DesignKit: React.FC = () => {
     }
   }, []);
 
+  useInitApi();
```

Comment:
why are we using `useInitApi` here if all calls to `makeToast` have been replaced?